graspy/utils/utils.py

```diff
@@ -253,6 +266,12 @@ def to_laplace(graph, form="DAD"):
         raise ValueError("Laplacian not implemented/defined for directed graphs")
 
     D_vec = np.sum(A, axis=0)
+    # regularize laplacian with parameter
+    # set to average degree
+    if form == "R-DAD":
+        if t == None:
```

Comment:
@j1c I think we want this kwarg, the paper just says that it's a good default but not the theoretical best thing. Thoughts?

Assuming we want it, @vikramc1 can you write typechecking and tests to show that the kwarg works and the typechecking works?

Reply:
We want the arg. I would name it something different than `t`. It is not an informative name. Maybe `regularizer` or something. Feel free to suggest a better name.

Reply:
added typechecking and tests for it